wfe/wfe.go

```diff
@@ -1627,6 +1638,62 @@ func (wfe *WebFrontEndImpl) makeChallenges(authz *core.Authorization, request *h
 	return nil
 }
 
+// validateReplacementOrder performs several sanity checks on the order to
+// determine if the order is a replacement of an existing order. If the order is
+// not a replacement or is a valid replacement, no problem will be returned.
+// Otherwise the caller will receive a problem.
+func (wfe *WebFrontEndImpl) validateReplacementOrder(newOrder *core.Order) *acme.ProblemDetails {
```

Comment:
```suggestion
func (wfe *WebFrontEndImpl) validateReplacementOrder(newOrder *core.Order) error {
```

Reply:
I was copying the style of `wfe.VerifyOrder` which is called prior to this method. Is there any reason I should diverge?